src/js/cypress/Browser.ts

```diff
@@ -219,6 +249,16 @@ export class Browser {
         return cy.get(selector).invoke("text");
     }
 
+    getEachElementTexts(selector: string) {
```

Comment:
I don't understand what this is for just form the correct selector

Reply:
This is not mine. Came from merging another branch